src/node/src/client.js

```diff
@@ -81,7 +81,7 @@ function _write(chunk, encoding, callback) {
   batch[grpc.opType.SEND_MESSAGE] = this.serialize(chunk);
   this.call.startBatch(batch, function(err, event) {
     if (err) {
```

Comment:
It looks like it. There's also a failure on Travis that may or may not be related to what I did, but I can't reproduce it in over 1000 runs on my local machine.